runtime/compiler/p/codegen/J9CodeGenerator.cpp

```diff
@@ -77,6 +77,12 @@ J9::Power::CodeGenerator::initialize()
       cg->setSupportsInlineConcurrentLinkedQueue();
       }
 
+   static char *disableInlineVectorizedMismatch = feGetEnv("TR_disableInlineVectorizedMismatch");
+   if (cg->getSupportsArrayCmp() && !disableInlineVectorizedMismatch)
```

Comment:
@bhavanisn I've only considered `arraycmp` support in enabling this acceleration on Power. Is there anything else to take into consideration here?

Reply:
Should be good enough